apps/video/test/marionette/lib/video.js

```diff
@@ -192,6 +195,66 @@ Video.prototype = {
   },
 
   /**
+   * Taps 'pause' button and checks that the counter is not progressing.
+   * returns false if the counter has progressed.
+   */
+  pausePlayback: function() {
+    this.client.helper.waitForElement(Video.Selector.playpauseButton).click();
+    var currentTime = this.getElapsedTimeSeconds();
+    
+    // wait for 2 seconds
+    this.client.executeAsyncScript(function () 
+      {setTimeout(function(){
+        marionetteScriptFinished();
+      }, 2000);
+
+    });
+    return this.getElapsedTimeSeconds() == currentTime;
```

Comment:
@russnicoletti I am confused, if  'getElapsedTimeSeconds' changes from line 203 to line 212 shouldn't this condition fail?